examples/chat2-reliable/chat.go

```diff
@@ -259,28 +378,46 @@ func (c *Chat) parseInput() {
 }
 
 func (c *Chat) SendMessage(line string) {
+	c.mutex.Lock()
+	defer c.mutex.Unlock()
+
+	c.lamportTimestamp++
```

Comment:
Seems like this is the second occurence of increment of the timestamp - that seems like one of them being redundant